### PR TITLE
Bump to python3

### PR DESCRIPTION
**Pull request by blackplane**
Hi, openstack-cli didn't work and bumping it to python3 did the trick. Not sure about the implications for terraform, etc.

### CODE REVIEW

**Comment by pblaas** [comment]
Hi @blackplane thank you for the PR. Could you elaborate what didn't work for with with openstack-cli? and did you use this container image to connect to CloudVPS Openstack or another Openstack platform?

File: openstack-cli.sh [hunk 1]
```diff
@@ -24,6 +24,6 @@ else
 	-e OS_PROJECT_NAME="$OS_PROJECT_NAME" \
 	-e OS_USERNAME="$OS_USERNAME" \
 	-e OS_PASSWORD="$OS_PASSWORD" \
-	docker.io/pblaas/openstack-cli:latest
+	blackplane/openstack-cli:latest
```
Comment (pblaas) [inline openstack-cli.sh:27]: I can't accept this change, please revert it back.
```suggestion
	docker.io/pblaas/openstack-cli:latest
```

**Comment by blackplane** [comment]
Sure, openstack-cli is dropping python 2.7 (e.g., see https://github.com/openstack/python-openstackclient/commit/f01a0f336c497584cc38e4396ee87282bdef1f5c)

I've used the container to access our own OpenStack instance.

**Review by pblaas — CHANGES_REQUESTED** [review]
Tested the python changes and look good to me! 
Except for the reference to the docker container in the cli command which is pointing to your container instead of mine.

**Pull request merged**